nautobot_chatops_ipfabric/models.py

```diff
@@ -0,0 +1,41 @@
+"""IP Fabric data models."""
+
+from datetime import datetime
+
+
+class Snapshot:
```

Comment:
a cosmetic change, could we change this filename to somethign else? in Nautobot plugins we use the `models.py` to define Django models